src/main/java/com/github/cameltooling/lsp/internal/instancemodel/OptionParamURIInstance.java

```diff
@@ -23,11 +23,16 @@
 import org.apache.camel.catalog.CamelCatalog;
 import org.eclipse.lsp4j.CompletionItem;
 
+import com.github.cameltooling.model.ComponentModel;
+import com.github.cameltooling.model.EndpointOptionModel;
+
 /**
  * For a Camel URI "timer:timerName?delay=10s", it represents "delay=10s"
  *
  */
 public class OptionParamURIInstance extends CamelUriElementInstance {
+
+	public static final String INVALID_URI_OPTION = "The parameter you entered is not a known parameter of the component for the used Camel version!"; 
```

Comment:
What do you think to include the name of the parameter which was searched for? it would help to give more context to user

Reply:
good idea